File: codeforces/Team.c
#include <stdio.h>

int main(){
    int n, cnt = 0;
    scanf("%d", &n);
    int P, V, T;
    for (int i = 0; i < n; i++){
        scanf("%d %d %d", &P, &V, &T);
        if (P == 1){
            if (V == 1 || T == 1){
                cnt++;
            }
        }else if(V == 1 && T == 1){
            cnt++;
        }
    }
    printf("%d", cnt);
    return 0;
}

File: codeforces/Team.js
var n = readline();
var cnt = 0;
 
for (var i = 0; i < n; i++){
    var arr = readline().split(" ");
    var sum = 0;
    for (var j = 0; j < arr.length; j++){
        if(arr[j] === '1'){
            sum++;
        }
    }
    if (sum >= 2){
        cnt += 1;
    }
}
 
print(cnt);